server/chat-commands/moderation.ts

```diff
@@ -36,9 +36,9 @@ export const commands: ChatCommands = {
 		}
 
 		if (!this.can('makeroom')) return false;
+		if (room.auth.has(toID(name))) return this.errorReply(`${name} is already a room owner.`);
```

Comment:
...this means the user has some auth, not that the user is an RO.

Reply:
oops i'm dense